packager/packager.cc

```diff
@@ -1029,22 +1021,17 @@ struct PackageInfoFinder {
             auto compatibilityAnnotation = compatibilityAnnotationLit->asString();
             if (compatibilityAnnotation != core::Names::strict() && compatibilityAnnotation != core::Names::legacy()) {
                 if (auto e = ctx.beginError(send.loc, core::errors::Packager::InvalidConfiguration)) {
-                    e.setHeader("Argument to `{}` must be either 'strict' or 'legacy'", send.fun.show(ctx));
+                    e.setHeader("Argument to `{}` can only be 'legacy'", send.fun.show(ctx));
```

Comment:
Up to you, but you could consider specializing this error if `fun` is `strict` with a custom message and/or autocorrect.